src/CodeGenerators/SettingsGen/Models/Attributes/ParameterAttribute.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+
+namespace Microsoft.Omex.CodeGenerators.SettingsGen.Models.Attributes
+{
+	/// <summary>
+	/// Attribute representing a parameter in the Settings.xml file
+	/// </summary>
+	[AttributeUsage(AttributeTargets.Property, AllowMultiple = false)]
+	public sealed class ParameterAttribute : Attribute
+	{
+		/// <summary>
+		/// Name of the parameter
+		/// </summary>
+		public string Name { get; set; } = string.Empty;
+
+		/// <summary>
+		/// Value of the parameter
+		/// </summary>
+		public string Value { get; set; } = string.Empty;
+
+		/// <summary>
+		/// MustOverride field of the Parameter element
+		/// </summary>
+		public string? MustOverride { get; set; }
```

Comment:
Sorry, for a lot of duplicated comments, but should not `MustOverride` and `IsEncrypted` be `bool` or `bool?`?

Reply:
Can't be nullable boolean but will convert to bool